docs/capabilities.md

```diff
@@ -52,6 +52,7 @@ Capability | Description
 |`appium:wdaStartupRetries`|Number of times to try to build and launch `WebDriverAgent` onto the device. Defaults to 2.|e.g., `4`|
 |`appium:wdaStartupRetryInterval`|Time, in ms, to wait between tries to build and launch `WebDriverAgent`. Defaults to 10000ms.|e.g., `20000`|
 |`appium:wdaLocalPort`|This value if specified, will be used to forward traffic from Mac host to real ios devices over USB. Default value is same as port number used by WDA on device.|e.g., `8100`|
+|`appium:wdaRemotePort`|This value if specified, will be used as the port number to start WDA HTTP server on the remote device. This is only relevant for real devices, because Simulator shares ports with its host. If `webDriverAgentUrl` is provided then it might be used to provide a hint for the remote port number if it differs from the default one. Default value is 8100.|e.g., `8100`|
```

Comment:
yes, this was my idea. If webDriverAgentUrl is provided then we don't know which remote port is used by WDA. We may only assume it is the same as the local one, but this might not always be the case

Reply:
Can you add the `wdaRemotePort`'s possibility in https://github.com/appium/appium-xcuitest-driver/blob/5a6eec054c11add9b1924add0251295ffc79816c/docs/attach-to-running-wda.md?plain=1#L8 ?

In case a user starts a WDA process with a custom port (to 8100) on a device, `wdaRemotePort` should help to coordinate the device's WDA port. This is not necessary if the user launches WDA without giving any custom port. (then WDA starts with 8100, so no problem)

Reply:
added